packages/asset-swapper/src/utils/market_operation_utils/pools_cache/balancer_v2_utils.ts

```diff
@@ -6,15 +6,15 @@ import { gql, request } from 'graphql-request';
 
 import { DEFAULT_WARNING_LOGGER } from '../../../constants';
 import { LogFunction } from '../../../types';
-import {
-    BALANCER_MAX_POOLS_FETCHED,
-    BALANCER_TOP_POOLS_FETCHED,
-    BALANCER_V2_SUBGRAPH_URL_BY_CHAIN,
-} from '../constants';
+import { BALANCER_MAX_POOLS_FETCHED, BALANCER_TOP_POOLS_FETCHED } from '../constants';
 
 import { parsePoolData } from './balancer_sor_v2';
 import { CacheValue, PoolsCache } from './pools_cache';
 
+const BEETHOVEN_X_SUBGRAPH_URL_BY_CHAIN = new Map<ChainId, string>([
+    [ChainId.Fantom, 'https://api.thegraph.com/subgraphs/name/beethovenxfi/beethovenx'],
```

Comment:
I'm on the fence about having the config/urls/addresses in files outside of `packages/asset-swapper/src/utils/market_operation_utils/constants.ts`. Do you have a preference?

Reply:
For the constants that are only used in a single place, I prefer those constants to be defined in the same file as it's used and not `export`ed. 

Ideally, if it's for sampling of a liquidity source, all related constants and logic (router addresses, pool info, gas schedule) are located in a single file specific to that source (when a new source is added, it should be mainly adding a single file vs touching multiple parts of the code base).

Let's discuss more on Slack!